flixel/system/replay/FrameRecord.hx

```diff
@@ -60,6 +60,9 @@ class FrameRecord
 	public function save():String
 	{
 		var output:String = frame + "k";
+	
+		if ((keys == null) && (mouse == null))
```

Comment:
I think this is a problem, if we don't save the frame, how does the playback know to wait during frames where no input occurred?